packages/react/src/Alert.js

```diff
@@ -17,14 +17,17 @@ library.add(
   faXmark
 );
 
-function Alert({ message, style }) {
+function Alert({style, variant, message, autohide, delay}) {
   const [showAlert, setShowAlert] = useState(true);
 
+  autohide && typeof autohide === "boolean" ? autohide : false;
```

Comment:
We can also set default props for `autohide` and `delay` to be `false` and `5000` respectively.